app/models/concerns/duplicate_fixable.rb

```diff
@@ -280,6 +280,14 @@ def fix_duplicate(key, user)
       return message unless status
 
       duplicate_record.clean_up_duplicate(self)
+
+      if is_a?(User)
```

Comment:
this stuff was being done on each applicable project and had a TODO to move it here